invenio_app_ils/ui/backoffice/src/pages/LoanDetails/index.js

```diff
@@ -16,4 +17,4 @@ export default connect(
     error: state.loanDetails.error,
   }),
   mapDispatchToProps
-)(withRouter(LoanDetails));
+)(withRouter(detailsClass));
```

Comment:
MINOR: `DetailsComponent` maybe?

Reply:
Yes, I was looking for a name, thank you, a bit more explicit though, LoanDetailsComponent etc ..